source/index.html.md

```diff
@@ -1,239 +1,674 @@
+max | The maximum value in a column | `max(p0)`
+
+## Operator Examples
+
+>1) <code>https://api.namara.io/v0/data_sets/057d7914-839e-4625-b8f8-2aa109f11e5a/data/en-3/aggregation?api_key={YOUR_API_KEY}&operation=avg(co2_emissions_g_km)&where=make="CADILLAC"</code>
+
+>2) <code>https://api.namara.io/v0/data_sets/057d7914-839e-4625-b8f8-2aa109f11e5a/data/en-3/aggregation?api_key={YOUR_API_KEY}&operation=min(co2_emissions_g_km)&where=make="CADILLAC"</code>
+
+>3) <code>https://api.namara.io/v0/data_sets/057d7914-839e-4625-b8f8-2aa109f11e5a/data/en-3/aggregation?api_key={YOUR_API_KEY}&operation=count(*)&where=make="CADILLAC"</code>
+
+1. Reveals the average CO<sub>2</sub> emissions of Cadillac vehicles
+
+2. Reveals the Cadillac vehicle with the least CO<sub>2</sub> emission
+
+3. Reveals the Cadillac vehicles that have emissions data
+
+# Query API
+
+While the Data API allows query behaviour over a single data set, the Query API allow broader control over the Namara data catalog.
+
+This API supports **NiQL** (the **N**amara **i**o **Q**uery **L**anguage), a language for viewing, aggregating, and joining data sets. It's very similar to SQL, so don't worry - no new textbooks.
+
+##<div class="colour-pill"><span class="get">GET</span> Meta</div>
+
+>https://api.namara.io/v0/query/meta
 
-let api = kittn.authorize('meowmeowmeow');
-let kittens = api.kittens.get();
 ```
+{
+  "query_limit_maximum": 250,
+  "formats": ["geojson", "csv", "json"],
+  "default_format": "json"
+}
+```
+
+This endpoint provides the meta information for querying this Namara instance. It includes the maximum limit per query, the supported formats, and the default format if none is specified. This information may differ, depending on which deployment of Namara you're using. 
+
+##<div class="colour-pill"><span class="post">POST</span> Query</div>
+
+>https://api.namara.io/v0/query.:format
 
-> The above command returns JSON structured like this:
+A call here dispatches a query and returns the result.
 
-```json
-[
+### Request
+
+Parameter | Type | Description
+--------- | ---- | -----------
+format | `string` | Query response format: `csv`, `json`, or `geojson` (default is `json`)
+project_id | `string` | Project ID to use for subscription lookup. If not provided, it is inferred from API Key
+organization_id | `string` | Organization ID to use for subscription lookup. If not provided, it is inferred from API Key
+query (required) | `string` | NiQL query string
+geojson_feature_key (may be required) | `string` | Property name to use as geometry when rendering `geojson`
+
+The <code>geojson_feature_key</code> is required if the requested format is <code>geojson</code> - otherwise, it is not needed.
+
+### Response
+
+### 200: OK
+
+Query executed successfully.
+
+<button class="see-json">json</button> 
+<button class="see-csv">csv</button> 
+<button class="see-geojson">geojson</button> 
+
+<div class="center-column response-json"></div>
+```
+json 
```

Comment:
These examples are great. Can we get them for the data API too? The only difference between them is that json isn't rendered in the results, so it would look like

```
[
  { ... },
  ...
]
```